packages/core/template_project/app/build.gradle

```diff
@@ -143,6 +143,17 @@ android {
 
         resValue "bool", "enableSiteSettingsShortcut", twaManifest.enableSiteSettingsShortcut
         resValue "string", "orientation", twaManifest.orientation
+
+        // This is the value of the screenOrientation for the LauncherActivity which determines the
+        // orientation of the splash screen. The "landscape" and "portrait" values from the TWA
+        // manifest directly maps ot "landscape" and "portrait" screenOrientation. Other values
+        // can't be directly mapped and are left as "unspecified". See more at:
```

Comment:
It feels a bit odd that `portrait-primary` and `portrait-secondary` can't be mapped to `portrait`.

Reply:
It may be possible, I just didn't look into it yet. I didn't really test his change on a a device / emulator yet.

Reply:
From the [Android Manifest](https://developer.android.com/guide/topics/manifest/activity-element#screen) and [web lockOrientation](https://developer.mozilla.org/en-US/docs/Web/API/Screen/lockOrientation], it seems we would map:

|Web Manifest|Android Manifest|
|----|:----|
|`portrait`|`userPortrait`|
|`portrait-primary`|`portrait`|
|`portrait-secondary`|`reversePortrait`|
|`landscape`|`userLandscape`|
|`landscape-primary`|`landscape`|
|`landscape-secondary`|`reverseLandscape`|

Does this make sense?

Reply:
Yeah, I think that makes sense.